src/log/log.c

```diff
@@ -2687,7 +2687,7 @@ __log_write_internal(WT_SESSION_IMPL *session, WT_ITEM *record, WT_LSN *lsnp, ui
             __wt_log_slot_free(session, myslot.slot);
     } else if (force) {
         /*
-         * If we are going to wait for this slot to get written, signal the wrlsn thread.
+         * If we are going to wait for this slot to get written, signal the log server thread.
```

Comment:
Years ago the code below this comment was :
```
if (conn->log_wrlsn_cond != NULL) {
	WT_ERR(__wt_cond_signal(session, conn->log_wrlsn_cond));
	__wt_yield();
}
```
by Susan. I guess the comment is forgotten to be fixed. @sueloverso Am I right ?

Reply:
Normally we would have a separate pull request to cover comment updates in another area of the code.  But I've already taken the trouble to verify this, so we'll include it.